src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/segment_image_from_no_negative_text_prompt_subtree.xml

```diff
@@ -32,31 +32,33 @@
         image="{image}"
         masks="{masks2d}"
         masks_visualization_topic="{masks_visualization_topic}"
+        opacity="0.500000"
+        bounding_box_detection_class="{prompts}"
```

Comment:
@pac48 Looks like the linter made a bunch of other changes but this is the salient change.